src/main/java/com/robosh/ejournal/constant/Query.java

```diff
@@ -0,0 +1,17 @@
+package com.robosh.ejournal.constant;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum Query {
+
+    CHECK_EMAIL_UNIQUE("SELECT count(*) from %s where '%s' = '%s'");
```

Comment:
why name is check email, but query is about select count of emails

Reply:
If count == 0, email is unique

Reply:
i understand but why dont you name it COUNT_OF_EMAILS?

Reply:
or even COUNT_OF_SOMETHING idk, because this query seems universal

Reply:
I thought about it but for me was better that variant

Reply:
Do we save custom queries in enum? Isn't it better to save them as constant in relative class? Or it's used in many occasions?

Reply:
there are many variants(enum, constant in class, interface, property, hardcode)